test_util/cluster.py

```diff
@@ -91,10 +89,7 @@ def __init__(
             'All cluster hosts require a private IP. hosts: {}'.format(repr(self.hosts))
         )
 
-        if self.provider == 'onprem':
-            self.check_ssh(self.hosts)
```

Comment:
No longer needed? Why did we check_ssh for all `hosts` onprem instead of just `masters` in the first place?